Client/Frontend/Reader/View/ReaderModeBarView.swift

```diff
@@ -45,10 +45,16 @@ protocol ReaderModeBarViewDelegate: AnyObject {
 }
 
 class ReaderModeBarView: UIView, AlphaDimmable, TopBottomInterchangeable, SearchBarLocationProvider {
+    private struct UX {
+        static let buttonWidth: CGFloat = 80
+    }
+
     weak var delegate: ReaderModeBarViewDelegate?
 
     var parent: UIStackView?
 
+    var contextStrokeColor: UIColor!
```

Comment:
I think we should use optional instead of force unwrap optional at the moment we are good because `applyTheme` is being called before the `draw` function is used so the color gets its value. My concern is that in  the future we might have a change in the flow and the color doesn't get initialized which will result in a crash using optional here and unwrapping in the guard we have in draw will be safer
```suggestion
    var contextStrokeColor: UIColor?
```